pkg/cmd/monitor/health/health.go

```diff
@@ -0,0 +1,142 @@
+package health
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"go.etcd.io/etcd/clientv3"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/connectivity"
+)
+
+const (
+	SerializedReadSingleTarget CheckName = "SerializedReadSingleTarget"
+	QuorumReadSingleTarget     CheckName = "QuorumReadSingleTarget"
+	GRPCReadySingleTarget      CheckName = "GRPCReadySingleTarget"
+	QuorumRead                 CheckName = "QuorumRead"
+
+	DefaultSlowRequestDuration = 5 * time.Second
```

Comment:
we log if request takes longer than DefaultSlowRequestDuration this could be dropped if it is felt to not be useful.

Reply:
> we log if request takes longer than DefaultSlowRequestDuration this could be dropped if it is felt to not be useful.

it's useful.  But I would expect it close to 500ms